python/lsst/daf/butler/core/quantum.py

```diff
@@ -19,13 +19,27 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-
-class Quantum(object):
-    """A discrete unit of work that may depend on one or more Datasets and produces one or more `Dataset`s.
-
-    Most Quanta will be executions of a particular `SuperTask`’s `runQuantum` method,
-    but they can also be used to represent discrete units of work performed manually
-    by human operators or other software agents.
+__all__ = ("Quantum",)
+
+
+class Quantum:
+    """A discrete unit of work that may depend on one or more Datasets and
+    produces one or more `Dataset`\ s.
+
+    Most Quanta will be executions of a particular `SuperTask`’s `runQuantum`
+    method, but they can also be used to represent discrete units of work
+    performed manually by human operators or other software agents.
+
+    Parameters
+    ----------
+    run : `Run`
+        Run this quantum is part of.
+    task : `SuperTask` or `str`, optional
+        Associated task information.
+    quantumId : ??, optional
+        ???
+    registryId : ??, optional
+        ???
```

Comment:
Are question marks intentional?

Reply:
They mean I have no idea what they are and need guidance.